genesis/utils/usda.py

```diff
@@ -30,14 +42,15 @@ def get_input_attribute_value(shader, input_name, input_type=None):
     return None, None
 
 
-def get_texture_image(image_path, zipfiles):
-    if zipfiles is None:
-        return np.array(Image.open(image_path.resolvedPath))
-    else:
-        return np.array(Image.open(io.BytesIO(zipfiles.GetFile(image_path.path))))
+def get_texture_image(image_path):
```

Comment:
Remove this entire function. It is useless for a single line of code that is straightforward.